tests/publiccloud/migration.pm

```diff
@@ -21,42 +21,117 @@ use utils;
 use publiccloud::utils;
 
 our $target_version = get_required_var('TARGET_VERSION');
-our $not_clean_vm = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
+our $not_clean_vm   = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
```

Comment:
you can revert this line

Reply:
not clear why to revert.. I have this logic further down

` # Include debug mode
    if ($not_clean_vm) {
        $instance->ssh_script_run(
            cmd => 'sudo touch /etc/sle-migration-service.yml' );
        $instance->ssh_script_run( cmd => 'echo \"verbose_migration: true\" | sudo tee -a /etc/sle-migration-service.yml');
        $instance->ssh_script_run( cmd => 'echo \"debug: true\" | sudo tee -a /etc/sle-migration-service.yml');
        $instance->ssh_script_run( cmd => 'sudo cat /etc/sle-migration-service.yml' );
        record_info( 'INFO', 'created sle-migration-service.yml configuration' );
    }
`

Reply:
because only change in this line is additional space before `=`

Reply:
ah ok , now I see . sorry didn't thought about perltidy . yes keep it than